include/deal.II/base/geometry_info.h

```diff
@@ -4768,7 +4768,7 @@ GeometryInfo<dim>::project_to_unit_cell(const Point<dim, Number> &q)
 {
   Point<dim, Number> p;
   for (unsigned int i = 0; i < dim; ++i)
-    p[i] = std::min(std::max(q[i], Number(0.)), Number(1.));
+    p[i] = std::clamp(q[i], Number(0.), Number(1.0));
```

Comment:
It seems as you need to revert this change, because the testers complain as follows:
```
/Users/runner/work/dealii/dealii/include/deal.II/base/geometry_info.h:4771:17: note: in instantiation of function template specialization 'std::clamp<dealii::VectorizedArray<double, 2>>' requested here
    p[i] = std::clamp(q[i], Number(0.), Number(1.0));
                ^
/Users/runner/work/dealii/dealii/include/deal.II/fe/mapping_q_internal.h:1011:34: note: in instantiation of function template specialization 'dealii::GeometryInfo<1>::project_to_unit_cell<dealii::VectorizedArray<double, 2>>' requested here
              GeometryInfo<dim>::project_to_unit_cell(result));
                                 ^
/Users/runner/work/dealii/dealii/source/fe/mapping_q.cc:682:37: note: in instantiation of function template specialization 'dealii::internal::MappingQImplementation::InverseQuadraticApproximation<1, 1>::compute<dealii::VectorizedArray<double, 2>>' requested here
              inverse_approximation.compute(p_vec),
```
In short, we call this function with `VectorizedArray<double>` arguments for which we provide overloads for `std::min()` and `std::max()`, but not for `std::clamp` (and in fact, the implementation for that class would use `min` and `max` as far as I can tell, as those should be the most efficient choices). In any case, we could provide such an overload if you're interested going into that route.

Reply:
Thank you so much for the advice! I will work on fixing that.